shared/Reusble.jsx

```diff
@@ -606,7 +614,7 @@ export function TrashButton({ handleDelete, fieldValues, index }) {
   return (
     <button
       type="button"
-      aria-label="delete this field"
+      // disabled={!fieldValues[index].id}
```

Comment:
wat? how is this connected?

Reply:
@oryrosin I'm adding a TODO for you to check it out latetr

Reply:
i dont see it on my machine... same branch on starting point